OsmDataCollector.py

```diff
@@ -11,7 +11,7 @@
 import mplleaflet
 
 DIR_PATH = 'files\\traces'
-NUMBER_OF_WANTED_FILES = 1  # This is the number of gpx files we download from osm.
+NUMBER_OF_WANTED_FILES = 5  # This is the number of gpx files we download from osm.
```

Comment:
I agree, because the user should have means of control of how many tracks the class will generate. I think the correct thing to do is to pass the number of wanted tracks instead of the number of wanted files (because it doesn't exposes inner implementation). However, doing it would require some logic changes to the class that might take time. what do you think?